agent/instrumentation/micrometer-1.0/src/main/java/io/opentelemetry/javaagent/instrumentation/micrometer/ai/AzureMonitorMeterRegistry.java

```diff
@@ -28,9 +28,12 @@ public class AzureMonitorMeterRegistry extends StepMeterRegistry {
   public static final AzureMonitorMeterRegistry INSTANCE =
       new AzureMonitorMeterRegistry(Clock.SYSTEM);
 
+  private final AzureMonitorRegistryConfig config;
+
   // visible for testing
   public AzureMonitorMeterRegistry(Clock clock) {
     super(new AzureMonitorRegistryConfig(), clock);
+    config = new AzureMonitorRegistryConfig();
```

Comment:
can't init instance variable before the supertype constructor. not sure a better way to init it, it will return the same AzureMonitorRegistryConfig(). I new it twice at line 35 and 36. please let me know if you have a better way.